pkg/kubernetes/persistentvolumeclaim/v1alpha1/kubernetes.go

```diff
@@ -116,39 +127,39 @@ func NewKubeClient(opts ...KubeclientBuildOption) *Kubeclient {
 	return k
 }
 
-// getClientOrCached returns either a new instance
+// getClientSetOrCached returns either a new instance
 // of kubernetes client or its cached copy
-func (k *Kubeclient) getClientOrCached() (*kubernetes.Clientset, error) {
+func (k *Kubeclient) getClientSetOrCached() (*kubernetes.Clientset, error) {
 	if k.clientset != nil {
 		return k.clientset, nil
 	}
 	c, err := k.getClientset()
 	if err != nil {
-		return nil, err
+		return nil, errors.Wrapf(err, "failed to get clientset")
 	}
 	k.clientset = c
 	return k.clientset, nil
 }
 
 // Get returns a pvc resource
 // instances present in kubernetes cluster
-func (k *Kubeclient) Get(name string, opts metav1.GetOptions) (*v1.PersistentVolumeClaim, error) {
+func (k *Kubeclient) Get(name string, opts metav1.GetOptions) (*corev1.PersistentVolumeClaim, error) {
 	if strings.TrimSpace(name) == "" {
 		return nil, errors.New("failed to get pvc: missing pvc name")
 	}
-	cli, err := k.getClientOrCached()
+	cli, err := k.getClientSetOrCached()
 	if err != nil {
-		return nil, err
+		return nil, errors.Wrapf(err, "failed to get pvc {%s}: failed to get clientset", name)
```

Comment:
nit: Let's remove the redundant error ` failed to get clientset` as `getClientSet` method is already returns the same error, which will eventually appended to the err.

Reply:
Done